aa_stripe/models.py

```diff
@@ -131,7 +131,11 @@ class StripeCoupon(StripeBasicModel):
     def __str__(self):
         return self.coupon_id
 
-    def save(self, *args, **kwargs):
+    def save(self, force_retrieve=False, *args, **kwargs):
+        """Save StripeCoupon
```

Comment:
new lines, please. If multiline comment it looks much better. Also, one line space between first line and the rest is needed.